kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1143,6 +1143,16 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(false)
 
+  val ENGINE_USER_ISOLATED_SPARK_SESSION: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.user.isolated.spark.session")
+      .doc("When set to false, if the engine is running in a group or server share level, " +
+        "all the JDBC/ODBC connections will be isolated against the user. Including: " +
+        "the temporary views, function registries, SQL configuration and the current database. " +
```

Comment:
If kyuubi supports shared temp view, udf is very good, but sharing current database, will it be confusing to the same user in different sessions?

Reply:
it's a part of context, I think user should aware of this ..